homeassistant/components/tado/binary_sensor.py

```diff
@@ -251,6 +257,11 @@ def _async_update_zone_data(self):
 
         elif self.zone_variable == "overlay":
             self._state = self._tado_zone_data.overlay_active
+            self._state_attributes = (
+                {"termination": self._tado_zone_data.overlay_termination_type}
+                if self._tado_zone_data.overlay_active
+                else {}
+            )
```

Comment:
The Home Assistant standard is to convert a ternary to an if block if it spans multiple lines.